r/00_install_packages.R

```diff
@@ -2,11 +2,13 @@
 
 ## Here we record all the packages used in the analyses
 
-pkgs <- c("data.table",
+pkgs <- c(
```

Comment:
not necessary of course - i think i did it out of habit without thinking - is it ok?

Reply:
Yeah I've left as is for now unless you have a strong reason for defining the mirror we use?